src/main/resources/messages_de.properties

```
@@ -711,6 +711,9 @@ settings.overtime.description=Die maximale Anzahl der Überstunden bestimmt, wie
 settings.overtime.overtimeActive=Überstundenverwaltung aktivieren
 settings.overtime.overtimeActive.true=aktivieren
 settings.overtime.overtimeActive.false=deaktivieren
+settings.overtime.overtimeReductionWithoutApplicationActive=Überstundenabbau ohne Antrag aktivieren
```

```suggestion
settings.overtime.overtimeReductionWithoutApplicationActive=Überstundenabbau ohne Urlaubsantrag aktivieren
```
Bin unsicher was das beste Wöörding ist:

* Antrag auf Überstundenabbau
* Urlaubsantrag mit Überstundenabbau

In der Fehlermeldung unten ist es dann "Überstundenabbau ist nicht erlaubt. Bitte stelle einen **Urlaubsantrag** dafür."